components/script/dom/htmltextareaelement.rs

```diff
@@ -100,7 +100,7 @@ impl LayoutHTMLTextAreaElementHelpers for LayoutDom<'_, HTMLTextAreaElement> {
             // placeholder is single line, but that's an unimportant detail.
             self.placeholder()
                 .replace("\r\n", "\n")
-                .replace("\r", "\n")
+                .replace('\r', "\n")
```

Comment:
Is this https://rust-lang.github.io/rust-clippy/master/index.html#single_char_pattern ?
But I guess `replace` accepts a str and not a char, just looking at the line above.